oneview_redfish_toolkit/api/service_root.py

```diff
@@ -56,9 +56,9 @@ def __init__(self, oneview_uuid):
         self.redfish["CompositionService"] = collections.OrderedDict()
         self.redfish["CompositionService"]["@odata.id"] = \
             "/redfish/v1/CompositionService"
-        # self.redfish["EventService"] = collections.OrderedDict()
-        # self.redfish["EventService"]["@odata.id"] = \
-        # "/redfish/v1/EventService"
+        self.redfish["EventService"] = collections.OrderedDict()
```

Comment:
Just realized that, if authentication mode is set to "conf", we should not add "EventService" endpoint, since it is not supported now.

Reply:
You mean, we should only include the EventService API when the authentication mode is set to `conf`, right? I will fix this by adding a test before registering the EventService blueprint in the `app.py`, ok?

Reply:
Yes, both changes.